openreview/venue/invitation.py

```diff
@@ -2878,7 +2878,20 @@ def set_ethics_stage_invitation(self):
                                     'type': 'boolean',
                                     'const': True
                                 }
-                            }
+                            },
+                            'readers': [venue_id, self.venue.get_ethics_chairs_id() ,self.venue.get_ethics_reviewers_id('${{4/id}/number}')]
```

Comment:
I checked with Sherry and she was fine if the flag was shown to all the readers of the note. 

I agree that authors shouldn't see the flag if the reviewers are not released to them yet but should we should the flag to the reviewers, AC, SACs too?

Reply:
Perhaps we should. I do not think we absolutely have to but we might as well